shell/platform/tizen/external_texture.h

```diff
@@ -20,18 +20,28 @@
 
 namespace flutter {
 
+typedef enum {
+  ExternalTextureGLExtention_None,
+  ExternalTextureGLExtention_EGL_TIZEN_image_native_surface,
+  ExternalTextureGLExtention_EGL_EXT_image_dma_buf_import
+} ExternalTextureGLExtention;
```

Comment:
Could you rename the enums by following https://google.github.io/styleguide/cppguide.html#Enumerator_Names?

I'm aware that the strings originally are defined somewhere else but you should follow the naming convention when converting them to enums.

My suggestion is:
```suggestion
typedef enum {
  kExternalTextureExtensionTypeNone,
  kExternalTextureExtensionTypeNativeSurface,
  kExternalTextureExtensionTypeDmaBuffer,
} ExternalTextureExtensionType;
```

Reply:
Thanks! 👍